### PR TITLE
airflow image upgrade

### PR DESCRIPTION
**Pull request by HazalCiplak**
https://github.com/elifesciences/data-hub-issues/issues/5

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -127,18 +127,35 @@ build: airflow-build
...
 ci-test-exclude-e2e: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm peerscout-dags-dev ./run_test.sh
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm peerscout-dags-dev ./run_test.sh
 
 ci-test-including-end2end: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm  test-client
-	$(DOCKER_COMPOSE_CI) down -v
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm  test-client
```
Comment (de-code) [inline Makefile:152]: I think this line is causing it to fail. We want to delegate to `end2end-test`.
i.e. `$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" end2end-test`

File: Makefile [hunk 2]
```diff
@@ -127,18 +127,35 @@ build: airflow-build
...
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 down -v
```
Comment (de-code) [inline Makefile:154]: This is the same issues as above. Here we want to delegate to `clean`. Although it doesn't seem to be shorter than what we had before.

File: Makefile [hunk 3]
```diff
@@ -127,18 +127,35 @@ build: airflow-build
...
+	$(DOCKER_COMPOSE) run --rm  test-client
+	$(MAKE) clean
+
 ci-build-dev:
 	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" build-dev
 
 ci-test-exclude-e2e: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm peerscout-dags-dev ./run_test.sh
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm peerscout-dags-dev ./run_test.sh
```
Comment (de-code) [inline Makefile:148]: Same as the other issues. In this case we could delegate to `test`, in which case we could probably delete `run_test.sh`.
Alternatively leave it as it was.

File: Makefile [hunk 4]
```diff
@@ -127,15 +127,27 @@ build: airflow-build
...
+clean:
+	$(DOCKER_COMPOSE) down -v
+
+airflow-initdb:
+	$(DOCKER_COMPOSE) run --rm  webserver initdb
+
+
+end2end-test:
+	$(MAKE) clean
+	$(MAKE) airflow-initdb
```
Comment (de-code) [inline Makefile:139]: do we still need `airflow-initdb` here if it is in `run_test.sh`?

**Review by de-code — APPROVED** [review]

File: Makefile (hunk 4 above)
Comment (HazalCiplak) [reply Makefile]: removed

**Pull request merged**